Description: Rewrite parts of the manpage
 Better explain what gptsync is, why it's needed.
Author: Julien BLACHE <jblache@debian.org>

Index: refit-0.14/gptsync/gptsync.8
===================================================================
--- refit-0.14.orig/gptsync/gptsync.8	2010-04-19 18:33:34.024931691 +0200
+++ refit-0.14/gptsync/gptsync.8	2010-04-19 18:33:32.938931385 +0200
@@ -1,4 +1,4 @@
-.TH "gptsync" 8 "2008 May 23" "Debian" "rEFIt"
+.TH "gptsync" 8 "2008 July 24" "Debian" "rEFIt"
 .SH NAME
 gptsync \- GPT partition table to MBR partition table synchronisation
 
@@ -7,11 +7,14 @@
 [\fI\-q\fR] \fIdevice\fR
 
 .SH DESCRIPTION
-Reads the GPT partition table on the device and synchronise content of
-MBR partition table on the device.  Useful for situations (as in
-mactel linux) where legacy operating systems require MBR partition
-table to function properly, while most other operating systems can
-work with GPT.
+.B gptsync
+reads the GPT partition table on \fIdevice\fR and synchronises the legacy
+MBR partition table with the GPT partition table. The MBR partition table
+can hold only 4 partitions.
+.PP
+Legacy bootloaders or operating systems that do not support GPT require an
+MBR partition table; this is typically the case when using GRUB or LILO on
+an Intel-based Apple machine.
 
 .SH OPTIONS
 .TP
@@ -19,6 +22,9 @@
 Do not print any message, do not prompt to modify the MBR.
 
 .SH AUTHOR
-Written by Christoph Pfisterer. This manual page contributed for Debian by
-Junichi Uekawa <dancer@debian.org>, but may be used for others.
+\fBgptsync\fR was written by Christoph Pfisterer.
+
+This manual page was written by Junichi Uekawa <dancer@debian.org> and
+Julien BLACHE <jblache@debian.org>, for the Debian Project (but may be used
+by others).
 
